echidna/scripts/dump_shrink.py

```diff
@@ -30,8 +30,8 @@ def main(args):
     """Smears energy and dumps spectra.
 
     Args:
-      args (Namespace): Container for arguments. See
-        >>> python dump_smeared_energy.py -h
+      args (Namespace): Container for arguments. See::
+        python dump_smeared_energy.py -h
```

Comment:
Just reviewing some PRs, one final change here, this line still doesn't get formatted correctly, I think it needs a blank line after `::`.

Otherwise this is good to be merged.